### PR TITLE
Set up component

### PR DESCRIPTION
**Pull request by swanie21**
# Purpose

Add Application component and styling 

# Inline Image Issue 

Thought it would be nice to have this boilerplate all set-up to render png/jpg/svg images. I'm trying to render local images, but haven't been successful. I Installed `url-loader` and `file-loader` and changed the `webpack.config.js` file to accommodate these loaders. There aren't any errors in the console, so the image file paths seem to be correct, but the image isn't appearing. Rendering static images works fine as you can see if you start up the server. These are the docs that I've been following:
[Rendering Local Images with React and Webpack](https://medium.com/@kathleeng.lopez/rendering-local-images-with-react-and-webpack-5f9d1fdb9b54#.cckaxtaq2)
[Dynamically Add Images React Webpack](http://stackoverflow.com/questions/32612912/dynamically-add-images-react-webpack/35454832?noredirect=1#comment56021739_32613874)
[Loading Images](https://survivejs.com/webpack/loading/images/)

### CODE REVIEW

File: lib/components/Application.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React, { Component } from 'react';
+// const HappyFace = require('../../public/img/happy-face.png');
```
Comment (swanie21) [inline lib/components/Application.js]: Unable to get this `HappyFace` image to render

Comment (Jeff-Duke) [reply lib/components/Application.js]: I copied the loaders section from the old repo and it's working now.  no idea what the issue was exactly with the way you had it configured but I'm able to see both the happy face and the celebrating guy now.  pull down the repo and make sure it's loading how you were wanting it to.  I think this is ready to merge now.

**Comment by swanie21** [comment]
# Purpose 
I added images in order to confirm that our loaders were working correctly. I included PNG and SVG formats. 

Additionally, I created a variables file for React's color scheme to help show that Sass is wired up and working correctly. 

This branch is ready to merge! 

# Next Steps
Include test files

**Comment by Jeff-Duke** [comment]
This looks good, merging.

**Pull request merged**